issue.go

```diff
@@ -32,13 +33,21 @@ type Issue struct {
 	Status       *IdName        `json:"status"`
 	Priority     *IdName        `json:"priority"`
 	Author       *IdName        `json:"author"`
+	FixedVersion *IdName        `json:"fixed_version"`
 	AssignedTo   *IdName        `json:"assigned_to"`
 	Notes        string         `json:"notes"`
 	StatusDate   string         `json:"status_date"`
 	CreatedOn    string         `json:"created_on"`
 	UpdatedOn    string         `json:"updated_on"`
 	CustomFields []*CustomField `json:"custom_fields,omitempty"`
 }
+type IssueFilter struct {
+	ProjectId    string
```

Comment:
I guess these tags are for json response, but this sturct 'IssueFilter' is not for json response, just use in argument of method.Would you need these?